### ISSUE TITLE
SCA: subscription renewal orders charge immediately without payment intent

### ISSUE THREAD

**Issue by madeincosmos** [woocommerce/woocommerce-gateway-stripe#931]
#### Affected ticket(s)
2210644-zen

#### What I expected
To use SCA on WooCommerce Subscription orders - both initial orders and renewals alike

#### What happened instead
I can see there's a payment intent created with the first parent subscription order in Stripe:

![Screenshot](https://cld.wthms.co/S8Ol4L+)
Screenshot: https://cld.wthms.co/S8Ol4L

However, subsequent renewal orders charge the customer immediately without using a payment intent:

![Screenshot](https://cld.wthms.co/1KVx1l+)
Screenshot: https://cld.wthms.co/1KVx1l

#### Steps to reproduce the issue
1. Create a daily subscription product,
2. Set up your site in EU and enable Stripe payments,
3. Purchase the subscription product and observe it creates a payment intent,
4. Wait until the subscription renews, or process renewal manually from the subscription page in WP Admin
5. Observe there's no payment intent this time.

User in 2210644-zen got the following message from Stripe support on that topic:

```
After reviewing the logs of your Dashboard here:

https://dashboard.stripe.com/logs?method[]=post&method[]=delete&direction[]=connect_in&direction[]=self

I do see that a Paymentintent was successfully created on June 21st, however all the payments we've received after that have been sent to us as charges. The Paymentintent creation can be seen within this log here:

[link edited]

While the logs after that are all charges, as shown here:

[link edited]

While reviewing this it does appear as though these are all coming from WooCommerce, so I would recommend reaching out to them to see why they have reverted back to sending charge requests instead of Paymentintents, as the Paymentintents would be the SCA ready way of processing. They should definitely be able to help out further and get this adjusted for you.
```

-------------------

- [ ] Issue assigned to next milestone.
- [ ] Issue assigned a priority (will be assessed by maintainers).

**Comment by jrodger** [comment]
I think this will be handled by the subscriptions work @DanReyLop?

Although I am wondering about your comment regarding grandfathering old subscriptions - https://github.com/woocommerce/woocommerce-gateway-stripe/pull/817#issuecomment-515781718. I wonder what the example subscription here would look like in that case?

**Comment by DanReyLop** [comment]
> I think this will be handled by the subscriptions work @DanReyLop?

Yes.

> Although I am wondering about your comment regarding grandfathering old subscriptions - #817 (comment). I wonder what the example subscription here would look like in that case?

An existing subscription would use a PaymentIntent with `off_session=true`, instead of a `Charge`. Stripe will do all the "grandfathering" logic behing the scenes, although there's always a chance that the customer will need to re-authenticate.

I've added this issue to milestone 4.3.0.

**Comment by iMazed** [comment]
Had a customer in chat worried about this, opened up a ticket so we can follow up: 2238230-zen

**Comment by DanReyLop** [comment]
We have created issues to track this work. They're all under the `4.3.0` release milestone: https://github.com/woocommerce/woocommerce-gateway-stripe/milestone/22

These are the main issues to look for: https://github.com/woocommerce/woocommerce-gateway-stripe/issues/938 https://github.com/woocommerce/woocommerce-gateway-stripe/issues/937 https://github.com/woocommerce/woocommerce-gateway-stripe/issues/936 https://github.com/woocommerce/woocommerce-gateway-stripe/issues/935

I'm gonna leave this issue open so we can notify you when it's fixed. In the meantime, there's nothing that can fix this problem (which only happens in test situations anyway). None of this problem will affect real users until September 14th, and by that time, we plan to have Stripe `4.3.0` implemented, tested, and released.

**Comment by v18** [comment]
Solved by #955
